### DISCUSSION TITLE
The indentation in cards cannot be displayed

### DISCUSSION THREAD

**Discussion by jacks935** [st3v3nmw/obsidian-spaced-repetition#909] [Q&A]
Hello author,spaced Repetition is a great plugin, but one issue that bothers me is that the card view seems to be operating in Obsidian's reading mode, where indents are not displayed. As a result, when reviewing, it's not possible to see the hierarchical relationship between pieces of knowledge. Could it be configured to use Obsidian's live preview mode instead?

**Comment by ronzulu** [comment]
Hi @jacks935 it would be helpful if you could post:

- A markdown file that exhibits the problem (please attach, not copy & paste)
- A screenshot showing the issue 

Thanks
Ronny

**Reply by jacks935** [reply]
![mmexport1711273325239.jpg](https://github.com/st3v3nmw/obsidian-spaced-repetition/assets/112944276/625e6eb0-ee18-45f1-a9a1-506de0e01743)![mmexport1711273328189.jpg](https://github.com/st3v3nmw/obsidian-spaced-repetition/assets/112944276/11be9d05-8fed-4ea3-8437-8b0f861c3b3f)
Is the card using a reading mode? However, under this reading mode, there is an issue where indents do not display properly.

**Comment by ronzulu** [comment]
Hi @jacks935 it would be helpful if you could post:

- A markdown file that exhibits the problem (please attach, not copy & paste)

**Comment by jacks935** [comment]
[Spaced Expetition test.md](https://github.com/st3v3nmw/obsidian-spaced-repetition/files/14757825/Spaced.Expetition.test.md)
![Screenshot_2024-03-26-19-47-29-87_51606159b24eff83e24a54116878fe3e](https://github.com/st3v3nmw/obsidian-spaced-repetition/assets/112944276/c82e847a-97a0-413a-b28a-058cffd2ecb6)
This is the display of my file
![Screenshot_2024-03-26-19-51-49-24_51606159b24eff83e24a54116878fe3e](https://github.com/st3v3nmw/obsidian-spaced-repetition/assets/112944276/c9f35925-2502-4732-86cc-fa3c66c5a2fb)
This is the display of the card, which lacks indentation compared to the file

**Comment by 4Source** [comment]
As mentioned here https://github.com/st3v3nmw/obsidian-spaced-repetition/issues/956#issuecomment-2156801772 this is the expected behavior of the obsidian team because this uses the markdown renderer and markdown doesn't support indented text except lists you can see this if you switch to reading mode in your note
